Sources/Common/DataModel/Line/index.d.ts

```diff
@@ -29,6 +32,18 @@ export interface vtkLine extends vtkCell {
 	 */
 	getCellDimension(): number;
 
+	/**
+	 * Get the list of orientations (a list of quat) for each point of the line.
+	 * Can be null if the line is not oriented
+	 */
+	getOrientations(): quat[] | null;
```

Comment:
Instances of `quat[] | null` ( or any `| null`) should be switched to `Nullable<quat[]>` (or `Nullable<Array<quat>>`) for consistency.